modules/taskbar.less

```diff
@@ -4,7 +4,7 @@
 	display: none;
 	z-index: 10001;
 	width: 50%;
-	left: 50%;
```

Comment:
What is the use of this `!important` declaration? This always raises a red flag for me, and testing on my dev shows that the `left` is already `50%`